allinonerpc/src/main.rs

```diff
@@ -0,0 +1,63 @@
+use crate::stream::BlockInfokStreamConnector;
+use crate::stream::EpochStreamConnector;
+use crate::stream::FullblockStreamConnector;
+use crate::stream::SlotStreamConnector;
+use tokio_stream::StreamExt;
+
+mod stream;
+
+const GRPC_URL: &str = "http://127.0.0.0:10000";
+
+#[tokio::main(flavor = "multi_thread", worker_threads = 8)]
+pub async fn main() -> anyhow::Result<()> {
```

Comment:
I like the idea of a centralized wiring + dependency injection;
we maybe should have multiple flavors of that (e.g.lite-rpc only for send_tx + tx confirm, no history, no full blocks) early to see if the architecture is decoupled enough to support that